do/common.go

```diff
@@ -0,0 +1,53 @@
+package do
```

Comment:
Taken that common functionality to an external file looks good, or at least much better than duplicating the code.
what do you think @andrewsykim ?

The file is lacking the boilerplate comments at the top
```
/*
Copyright 2017 DigitalOcean

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/
```
(which also could be checked at the ci script @andrewsykim )

Reply:
ahh, good find, I'll be sure to add that to CI soon!

Reply:
I don't love the file name `common.go` but I think it's fine for now :)